docs/metrics.asciidoc

```diff
@@ -0,0 +1,102 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
```

Comment:
`https://www.elastic.co/guide/en/apm/agent/java` should probably be `https://www.elastic.co/guide/en/apm/agent/dotnet`, right? :)